smg-automotive/components-pkg
# Group label

References [jira ticket](https://autoricardo.atlassian.net/browse/DM-731)

## Motivation and context

The goal is to create vehicle group data component according to the design https://www.figma.com/file/QYphttx4oaLD2Sd2L50meg/AS24-Detail?node-id=2821%3A134703&t=8ZZh2GMduWgtwUXd-0

## Before

No data group component.

## After

Having data group component.

## How to test
Go to [storybook](https://group-label.components-pkg.branch.autoscout24.ch/?path=/story/components-data-display-vehicledatagroup--group-label&args=w:500px)
Check component and compare with design.
